packages/rspack/module.d.ts

```diff
@@ -169,6 +171,36 @@ declare namespace Rspack {
 		id: string | number;
 		(dependency: string): unknown;
 	}
+
+	interface Module {
```

Comment:
There is also a `filename` property for the webpack Modules (see https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/webpack-env/index.d.ts#L74)

but I do not see this property with rspack